api/templates/design.go

```diff
@@ -349,17 +349,22 @@ var _ = Service("templates", func() {
 // Error -------------------------------------------------------------------------------------------------------
 
 var GenericErrorType = Type("GenericError", func() {
+	Description("Generic error")
 	Attribute("statusCode", Int, "HTTP status code.", func() {
+		Meta("struct:tag:json", "statusCode")
 		Example(500)
```

Comment:
V `design.go` mame `GenericError`, pouziva sa ako layout pre vsetky chyby.
Chcem ho pouzit aj pre neocakavane chyby, teda tie co nie su definovane v `design`.

Na to si potrebujem definovat nazvy JSON fieldov.
Ked Goa to riesi oddelene od tejto struktury, ... ale ja ju chcem iba serializovat, bez potreby riesit dalsi kod.